src/components/block/ProgramCard/ProgramCard.js

```diff
@@ -0,0 +1,29 @@
+import PropTypes from "prop-types";
+import React from "react";
+import s from "./ProgramCard.module.css";
+
+const ProgramCard = ({ image, alt, title, description }) => (
+  <div className={s.programCardContainer}>
+    <div>
+      <img className={s.img} src={image} alt={alt} />
+    </div>
+    <div className={s.title}>{title}</div>
+    <div className={s.description}>{description}</div>
+  </div>
+);
+
+ProgramCard.propTypes = {
+  image: PropTypes.string,
+  alt: PropTypes.string,
+  title: PropTypes.string,
+  description: PropTypes.string,
+};
+
+ProgramCard.defaultProps = {
+  image: PropTypes.string.isRequired,
+  alt: PropTypes.string.isRequired,
+  title: PropTypes.string.isRequired,
+  description: PropTypes.string.isRequired,
+};
```

Comment:
I think I was probably a bit unclear with my previous comment about this. `defaultProps` is only used to set the default value of a property when it is not passed in by the caller. In other words, the values of the default props should be of the same type that is require, such as an actual placeholder string.

When you want to mark a prop type as required, it should go in the prop type definitions under the `.propTypes` property. When a prop is required, then you do not need to set a `defaultProp` for it, since it's invalid to instantiate the component without all of its required props set.

You'll want to make a change that looks like this:

```suggestion
ProgramCard.propTypes = {
  image: PropTypes.string.isRequired,
  alt: PropTypes.string.isRequired,
  title: PropTypes.string.isRequired,
  description: PropTypes.string.isRequired,
};
```